codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
+  status:
+    project:
+      default:
+        informational: true
+    patch:
+      default:
+        informational: true
```

Comment:
Same comment as in https://github.com/gchq/gaffer-experimental/pull/209:
Missing newline

Reply:
Added